schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-distribution.xsl

```diff
@@ -110,6 +110,15 @@
     </foaf:page>
   </xsl:template>
 
+  <xsl:template mode="iso19115-3-to-dcat"
+                match="mdb:distributionInfo/mrd:MD_Distribution/mrd:distributionFormat">
+    <dcat:distribution>
```

Comment:
Unless I'm really wrong, this element will be invalid. See https://semiceu.github.io/DCAT-AP/releases/3.0.0/#Distribution, access URL is required.

I think what @fxprunayre meant is to modify the existing template to process the online resources, to use the  transferOptions format if exists, otherwise use the distribution ones as you did in the original code.

> I would rather add an additional rule https://github.com/geonetwork/core-geonetwork/blob/main/schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-distribution.xsl#L336-L339 and gives priority to transferOptions format if any and then fallback on distribution ones.

Reply:
Thanks  a lot ! We did not know about it. Back to draft / analysis, ;-) !